src/grpc_generated/javascript/client.js

```diff
@@ -0,0 +1,101 @@
+const grpc = require('@grpc/grpc-js');
+const protoLoader = require('@grpc/proto-loader');
+const util = require('util');
+const PROTO_PATH = __dirname + '/proto/grpc_service.proto';
+const PROTO_IMPORT_PATH = __dirname + '/proto'
+
+const GRPCServicePackageDefinition = protoLoader.loadSync(PROTO_PATH, {
+    includeDirs: [PROTO_IMPORT_PATH],
+    keepCase: true,
+    longs: String,
+    enums: String,
+    defaults: true,
+    oneofs: true
+});
+
+function BufferToInt32Array(buf) {
+    newArray = new Int32Array(buf.byteLength / 4)
+    dv = new DataView(buf.buffer)
+    for (let i = 0; i < newArray.length; i++) {
+        newArray[i] = dv.getInt32(buf.byteOffset + (i * 4), littleEndian = true);
+    }
+    return newArray
+}
+
+const inference = grpc.loadPackageDefinition(GRPCServicePackageDefinition).inference;
+
+async function main() {
+    const argv = process.argv.slice(2)
+    const host = argv.length > 0 ? argv[0] : "localhost";
+    const port = argv.length > 1 ? argv[1] : "8001";
+
+    const model_name = "simple";
+    const model_version = "";
+    const batch_size = 1
+    const dimension = 16
+
+    const client = new inference.GRPCInferenceService(host + ':' + port, grpc.credentials.createInsecure());
+
+
+    const serverLive = util.promisify(client.serverLive).bind(client);
+    const serverReady = util.promisify(client.serverReady).bind(client);
+    const modelMetadata = util.promisify(client.modelMetadata).bind(client);
+    const modelInfer = util.promisify(client.modelInfer).bind(client);
+
+    serverLiveResponse = await serverLive({});
+    console.log("Triton Health - Live:", serverLiveResponse.live);
```

Comment:
Instead if just logging can you check if the server is live? If not, then exit with a non-zero status. No point in proceeding with the example code.